README.md

```diff
@@ -8,12 +8,8 @@
 
 The recommended method of installation is [through composer](http://getcomposer.org).
 
-```JSON
-{
-    "require": {
-        "phergie/phergie-irc-plugin-react-bridge": "dev-master"
-    }
-}
+```
+composer require phergie/phergie-irc-plugin-react-bridge:dev-master
```

Comment:
It doesn't seem to matter? https://gist.github.com/sitedyno/e8030323baa016d8f723f731c0b8f5d1

I also added a note about this plugin requiring an older version of phergie-irc-bot-react.